<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>490</width>
      <height>247</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Algerian</family>
           <pointsize>72</pointsize>
          </font>
         </property>
         <property name="text">
          <string>SIFABUL</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string>color: #FF69B4;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Usuario</string>
               </property>
               <property name="styleSheet">
                <string>color: #FF69B4;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Contraseña</string>
               </property>
               <property name="styleSheet">
                <string>color: #FF69B4;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLineEdit" name="UserName"/>
             </item>
             <item>
              <widget class="QLineEdit" name="Password"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="boton1">
             <property name="text">
              <string>Borrar</string>
             </property>
             <property name="styleSheet">
              <string>background-color: #FF69B4; color: white;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BotonOk">
             <property name="text">
              <string>Aceptar</string>
             </property>
             <property name="styleSheet">
              <string>background-color: #FF69B4; color: white;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>SISTEMA DE FICHAJES UNIVERSIDAD DE LEÓN</string>
       </property>
       <property name="styleSheet">
        <string>color: #FF69B4;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
